Sources/XCTest/XCTestCase.swift

```diff
@@ -8,12 +8,58 @@
 //
 //
 //  XCTestCase.swift
-//  Base protocol (and extension with default methods) for test cases
+//  Base class for test cases
 //
 
-public protocol XCTestCase : XCTestCaseProvider {
-    func setUp()
-    func tearDown()
+public typealias XCTestMethod = XCTestCase throws -> Void
```

Comment:
OK, so call me crazy, but I'd love to see documentation on anything marked `public`. I'd be open to hearing dissenting opinions, or even deferring adding documentation until a later pull request.